src/extensions/extensionsHelper.ts

```diff
@@ -834,13 +834,17 @@ export async function diagnoseAndFixProject(options: any): Promise<void> {
  * 2. Infer "latest" as the version if not provided
  */
 export async function canonicalizeRefInput(extensionName: string): Promise<string> {
```

Comment:
the word "name" has lost its value (at least to me...) consider naming this `input` and the variable `inferredRef`